terraform/projects/app-ecs-services/config/alerts/alerts.yml

```diff
@@ -55,3 +56,15 @@ groups:
     annotations:
         summary: "Service is approaching capacity."
         description: "The service name is {{ $labels.job }}. The URL experiencing the issue is {{ $labels.instance }}"
+
+# Just for testing, should be removed after testing complete on staging and production
```

Comment:
this needs to be removed

Reply:
Yes, will remove after testing on staging and production, or could otherwise trigger `NoFileSdTargets`